app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
The `Person` class is missing the `wife` and `husband` attributes. These attributes are used in the `create_person_list` function, so they should be defined in the `Person` class to avoid runtime errors.